system-int-test/test/cmr/system_int_test/search/collection_sorting_search_test.clj

```diff
@@ -444,3 +465,37 @@
 
          ["-ongoing" "entry-title"] [coll2 coll4 coll5 coll6 coll7 coll3 coll1]
          ["ongoing" "entry-title"] [coll1 coll3 coll2 coll4 coll5 coll6 coll7])))
+
+(deftest collection-usage-score-sorting-test
+  (let [make-named-collection (fn [cname]
+                                (d/ingest
+                                  "PROV1"
+                                  (dc/collection
+                                    {:entry-title cname
+                                     :short-name cname
+                                     :long-name (str cname "-long")
+                                     :version-id "1"})))
+        c1 (make-named-collection "a")
```

Comment:
You might want to add a collection for which there is no usage data.

Reply:
It is assigned min/max int depending on the query so unused actually have the highest usage when sorting in an ascending fashion and worst usage when sorting in descending. This results in collections without an explicit usage always being last in the list.